CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(screensaver.stars)
 
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
`cmake_minimum_required(...)` should always be the first line, even before `project(...)`.

Since you're touching it, please fix that.

Reply:
Is done, I do it then also on the others after they becomes changes.

Reply:
Thanks! :+1: